src/kernels/kernelWorkingFolder.node.ts

```diff
@@ -36,7 +36,11 @@ export class KernelWorkingFolder {
         let suggestedDir = await this.calculateWorkingDirectory(kernel.resourceUri);
         if (suggestedDir && (await this.fs.exists(suggestedDir))) {
             return suggestedDir;
-        } else if (kernel.resourceUri && (await this.fs.exists(kernel.resourceUri))) {
+        } else if (
+            kernel.resourceUri &&
+            kernel.resourceUri.scheme !== 'untitled' &&
```

Comment:
Debt = we weren't checking if it was untitled, 
No point chekcing if untitled files exist on disc or not, as the answer is obvious